dist/index.html

```diff
@@ -1,34 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon" />
+    <link rel="stylesheet" href="style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Art Gallery</title>
 <script defer src="main.js"></script></head>
-    <body>
-        <!-- ***********************  PAGE OVERALL CONTENT WRAPPER *********************** -->
-        <div class="container">
-      
-          <!-- ***********************  PAGE HEADER *********************** -->
-          <header class="header"></header>
-          <!-- ***********************  END OF PAGE HEADER *********************** -->
-      
-          <!-- ***********************  PAGE MAIN *********************** -->
-          <main class="main">
-      
-          </main>
-          <!-- ***********************  END OF PAGE MAIN *********************** -->
-      
-          <!-- ***********************  PAGE FOOTER *********************** -->
-          <footer class="footer">
-            <div class="footer__wrapper">
-             
-            </div>
-          </footer>
-          <!-- ***********************  END OF PAGE FOOTER *********************** -->
-      
+<body>
+  <div class="containerCommentsPopUp">
+  </div>
+  <div class="reservationPopup">
+  </div>
+    <header>
+      <div>
+        <img
+          src="https://raw.githubusercontent.com/fmanimashaun/artwork-listing-app/combined_soln/src/img/logo.png"
+          class="home-logo"
+          alt=""
+          sizes=""
+          srcset=""
+        />
+        <span class="fd-number">Gallery (<span class="fd-count">0</span>)</span>
+        <span class="fd-categories">Painting</span>
+        <span class="fd-categories">Shop</span>
+        <div class="artworks">
```

Comment:
- [ ] You did great making the design, which I'll applaud you for, but I strongly suggest we take the view seriously from a user perspective:

- The text fonts look too small, and I think we can bold them a little for better user view 👍 
- From the wireframe, you can see how visible the popup is relative to the reflective background. I advise adjusting the saturation or making the popup background entirely white. again, this is for the best web accessibility 👍

## Your views:
![image](https://user-images.githubusercontent.com/50990925/218246833-faf59823-2c72-47b4-8c2b-ceb6b3bf3cf1.png)
 
 ## Wireframe with clarity:
 
![image](https://user-images.githubusercontent.com/50990925/218246851-be65fe63-39cd-43b3-8271-ee4fbf509c4f.png)

Reply:
Well resolved ✔️